.github/workflows/release.yaml

```diff
@@ -25,10 +34,24 @@ jobs:
           echo -n "git_info=" >> $GITHUB_OUTPUT
           bash -c "set -eu -o pipefail; source ./scripts-common.sh; git_info" >> $GITHUB_OUTPUT
       - name: install golang
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
         with:
           go-version-file: 'go.mod'
-          cache: true
+
+      # TODO: Check if we need this step from neonvm/
+      #
+      # This step is analogue to the `make generate` and `make manifests` steps in the Makefile.
+      # Since we commit all generated files, we don't need to run these steps again.
+      #
+      # - name: generate code and k8s CRDs
+      #   run: |
+      #     git clone --branch=v${{ env.CODE_GENERATOR_VERSION }} --depth=1 https://github.com/kubernetes/code-generator.git ${RUNNER_TEMP}/code-generator
+      #     go install sigs.k8s.io/controller-tools/cmd/controller-gen@v${{ env.CONTROLLER_TOOLS_VERSION }}
+      #     ${RUNNER_TEMP}/code-generator/generate-groups.sh "deepcopy,client,informer,lister" github.com/neondatabase/neonvm/client github.com/neondatabase/neonvm/apis neonvm:v1 --go-header-file hack/boilerplate.go.txt
+      #     controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/common/crd/bases output:rbac:artifacts:config=config/common/rbac output:webhook:artifacts:config=config/common/webhook
```

Comment:
In NeonVM we have this code in [`release.yaml`](https://github.com/neondatabase/neonvm/blob/d9ce98c5d7e949ff25049d9b6dab7f2142184290/.github/workflows/release.yaml#L34-L39), but I far as I understand we commit all autogenerated code to the repo, so this step shouldn't be necessary, am I missing something?
/cc @cicdteam

Reply:
seems yes, we can skip code generating in GA